libs/ett/auth/feature-login/src/lib/ett-authentication-link.component.spec.ts

```diff
@@ -17,7 +17,7 @@
 import { APP_BASE_HREF } from '@angular/common';
 import { TestBed } from '@angular/core/testing';
 import {
-  AuthOidcHttp,
+  AuthHttp,
```

Comment:
suggestion/question:

Should we use explicit name for services, "AuthHttp" vs. "AuthHttpService"?

Reply:
Duplicate of https://github.com/Energinet-DataHub/greenforce-frontend/pull/280#discussion_r788604789